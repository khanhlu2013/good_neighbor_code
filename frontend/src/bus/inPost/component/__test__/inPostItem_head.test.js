import React from "react";
import { shallow } from "enzyme";
import InPostItemHead from "../inPostItem_head";

describe("inPostItem_head", () => {
  it("when render -> match snapshot", () => {
    const wrap = shallow(
      <InPostItemHead
        postUserName="Lu"
        postUserEmail="abc@efg.com"
        dateCreate={new Date(2000, 1, 1)}
      />
    );

    expect(wrap).toMatchSnapshot();
  });
});
