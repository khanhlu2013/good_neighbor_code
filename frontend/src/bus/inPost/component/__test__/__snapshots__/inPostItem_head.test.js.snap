// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inPostItem_head when render -> match snapshot 1`] = `
<styled.div>
  <styled.div>
    <span
      className="text-primary font-weight-bold"
    >
      Lu
    </span>
    <span
      className="text-secondary font-weight-light"
    >
       abc@efg.com
    </span>
  </styled.div>
  <div
    className="text-secondary font-weight-light"
  >
    post on: xx/xx/xx
  </div>
</styled.div>
`;
